code/game/gamemodes/wizard/wizard.dm

```diff
@@ -378,9 +439,24 @@ Made a proc so this is not repeated 14 (or more) times.*/
 						if(I.icon_state == "wizard")
 							wizard_mind.current.client.images -= I
 
+		for(var/datum/mind/apprentice_mind in apprentices)
+			if(apprentice_mind.current)
+				if(apprentice_mind.current.client)
+					for(var/image/I in apprentice_mind.current.client.images)
+						if(I.icon_state == "wizard" || I.icon_state == "apprentice")
```

Comment:
This is a really poor way of doing it. Not only are you introducing a bug if the icon states are ever changed in the file, but it's just lazy. Use a list of images to keep track of the wizard icon ones.